packages/eslint-plugin/docs/rules/prefer-readonly.md

```diff
@@ -22,6 +22,7 @@ class Container {
   // These member variables could be marked as readonly
   private neverModifiedMember = true;
   private onlyModifiedInConstructor: number;
+  #neverModifiedPrivateField = 3;
```

Comment:
[Docs] Earlier in the page, it says:

> Member variables with the privacy `private` are never permitted to be modified outside of their declaring class. If that class never modifies their value, they may safely be marked as `readonly`.
>
> This rule reports on private members are marked as `readonly` if they're never modified outside of the constructor.

Can you add a mention of `#` privates somewhere in there too? The explanation is no longer accurate now that they're also considered by the rule.